webhooks/run_test.go

```diff
@@ -61,13 +61,15 @@ func run(ctx context.Context, cfg *rest.Config, scheme *runtime.Scheme, opts *en
 		},
 	})
 
-	go func(ctx context.Context, backfillComplete chan struct{}) {
-		podMutator.BackfillPermissions(ctx, backfillComplete)
-	}(ctx, backfillComplete)
+	errChan := make(chan error, 1)
+	go func() {
+		if err := podMutator.BackfillPermissions(ctx); err != nil {
+			errChan <- err
+		}
+	}()
+	go func() {
+		errChan <- mgr.Start(ctx)
+	}()
```

Comment:
What about the case where mgr.Start returns nil first then the BackfillPermissions returns an error? It'd be an unhandled error, right?

Reply:
Start won't return until nil the context is closed, the podMutator.BackfillPermissions will always fail before this (or its associated test)

Reply:
Can a comment be added in-code to explain this

Reply:
added comment